docs/model_zoo/action_recognition.rst

```diff
@@ -0,0 +1,61 @@
+.. _gluoncv-model-zoo-action_recognition:
+
+Action Recognition
+====================
+
+.. role:: greytag
+
+Table of pre-trained models for video action recognition and their performance.
+
+.. hint::
+
+  Training commands work with this script:
+  :download:`Download train_recognizer.py<../../scripts/action-recognition/train_recognizer.py>`
+
+  A model can have differently trained parameters with different hashtags.
+  Parameters with :greytag:`a grey name` can be downloaded by passing the corresponding hashtag.
+
+  - Download default pretrained weights: ``net = get_model('inceptionv3_ucf101', pretrained=True)``
+
+  - Download weights given a hashtag: ``net = get_model('inceptionv3_ucf101', pretrained='0c453da8')``
+
+  The test script :download:`Download test_recognizer.py<../../scripts/action-recognition/test_recognizer.py>` can be used for
+  evaluating the models.
+
+.. role:: tag
+
+UCF101 Dataset
+--------------
+
+The following table lists pre-trained models trained on UCF101.
+
+.. note::
+
+  Our pre-trained models reproduce results from "Temporal Segment Networks" [2]_ . Please check the reference paper for further information.
+
+  The top-1 accuracy number shown below is for official split 1 of UCF101 dataset, not the average of 3 splits.
+
+  ``InceptionV3`` is trained and evaluated with input size of 299x299.
+
+.. table::
+   :widths: 55 10 10 15 10
+
++---------------------------------------------+-----------+-----------+------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
indent on the table to make `widths` effective: https://stackoverflow.com/questions/48083214/setting-column-width-with-restructuredtext-grid-tables

Reply:
Thanks a lot for pointing it out. Resolved.